pom.xml

```diff
@@ -575,11 +575,6 @@
         <artifactId>dom4j</artifactId>
         <version>2.1.4</version>
       </dependency>
-      <dependency>
```

Comment:
Would probably be good to also add an entry to make it a banned dependency.

Reply:
I've added an exclude rule, extending the existing rule.

Reply:
I had forgotten about the now useless `enforce-jdom` rule, indeed.